README.md

```diff
@@ -20,10 +20,11 @@ npm install react-redux-notify --save
 
 After which you can import the component and the default styles for use:
 ```javascript
-import Notify from 'react-redux-notify';
```

Comment:
The `import Notiify` is not required here, I think.  And the syntax is incorrect anyway: it's actually importing the reducer, not the component!